common.yml

```diff
@@ -65,6 +65,27 @@
         with_sequence: 0-{{ (oc_instackenv_content.nodes|length - 1) }}
         register: host_list
 
+      - name: remove existing bad-hosts file
+        file:
+            path: "~/jetpack/badfish/bad-hosts"
+            state: absent
+
+      - name: prepare dell hosts for badfish
+        shell: echo "mgmt-{{ item[1].stdout }}" >> ~/jetpack/badfish/bad-hosts
+        when: vendors is defined and item[0] == "dell"
+        with_together:
+          - "{{ vendors }}"
+          - "{{ host_list.results }}"
+
+      - name: remove existing badfish image
```

Comment:
If the image is up to date, I don't think we need to remove it. It will simply lead to additional time being spent. Maybe you can change this step to remove the existing image only if the image is not the latest version.

Reply:
The existing image will contain list of dell hosts from the previous deployment. so, pulling it freshly will add the new hosts.
refer to jetpack/badfish/Dockerfile

Reply:
Ack